doc/source/whatsnew/v1.6.0.rst

```diff
@@ -39,10 +39,44 @@ Notable bug fixes
 
 These are bug fixes that might have notable behavior changes.
 
-.. _whatsnew_160.notable_bug_fixes.notable_bug_fix1:
+.. _whatsnew_160.notable_bug_fixes.notable_bug_frame_at_new_column_with_slice:
+
+Using :meth:`DataFrame.at` raises when trying to add a new column with multiple values
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*Old behavior*
+
+.. code-block:: ipython
+    In [1]: import pandas as pd
```

Comment:
You need a blank line here between `.. code-block:: ipython` and the first code line (this is the reason that the doc build is failing)

Reply:
In general, it might also be sufficient to just note this as a one-liner, if we consider this as a not-that-notable bug fix (although it did work "correctly" in the past, it was just not failing and doing too much for what we want `at` to do)